### DISCUSSION TITLE
invalid input syntax for type bigint when passing uuid

### DISCUSSION THREAD

**Discussion by XamHans** [supabase/supabase#11891] [Questions] [answered]
Hi there,

I want to query a specific entry by its id (uuid) the code looks like this:

`   const { data, error } = await supabaseClient
        .from(table)
        .select(`* ${relation ?? ''}`)
        .eq('id', id)
        .single()`

The request has this info as payload:
`select: 
*,profiles(*)
id: 
eq.4fe56cc4-0758-4bf5-870c-00b6a461ffd8`

But all I get is this weird error 

`code: "22P02", details: null, hint: null,…}
code
: 
"22P02"
details
: 
null
hint
: 
null
message
: 
"invalid input syntax for type bigint: \"4fe56cc4-0758-4bf5-870c-00b6a461ffd8\""`

The error occurred after re-creating the posts table with uuid column (which was previously bigint8), similar to issue #11336 

Restarting the database fixed the issue

**Comment by GaryAustin1** [comment, marked as answer]
https://github.com/supabase/supabase/issues/11714
Probably was this issue which is in PostgREST.

**Reply by steve-chavez** [reply]
Btw this was already solved and even existing projects were patched(no need for users to pause/unpause, but it might take a while to get the update in some cases).

**Comment by webinuse** [comment]
I have literally the same problem, also pretty similar situation, and it's not on the self-hosted Supabase. Any solutions for that?

**Reply by GaryAustin1** [reply]
Did you read the link?   Did you restart your hosted instance?

**Reply by webinuse** [reply]
As I mentioned in my comment it is NOT self-hosted instance, it is on Supabase. I couldn't find a way to restart Supabase. 😁

But I solved it by recreating database.

**Comment by GaryAustin1** [comment]
Sorry for the terminology confusion , to me hosted is the Supabase hosted platforms.  
for the future restart is on the general settings page.
